vlmeval/config.py

```diff
@@ -17,6 +17,11 @@
     'llava_v1_7b': 'Please set your local path to LLaVA-7B-v1.1 here, the model weight is obtained by merging LLaVA delta weight based on vicuna-7b-v1.1 in https://github.com/haotian-liu/LLaVA/blob/main/docs/MODEL_ZOO.md with vicuna-7b-v1.1. '
 }
 
+emu_model_path_map={
```

Comment:
You can remove this and place the model_map in emu.py, no need to add a model_path_map arg.